layers/core_checks/cc_ray_tracing.cpp

```diff
@@ -2381,7 +2373,434 @@ bool CoreChecks::ValidateBuildPartitionedAccelerationStructureInfoNV(
 
     if (SafeModulo(build_info.srcInfosCount, 4) != 0) {
         skip |= LogError("VUID-VkBuildPartitionedAccelerationStructureInfoNV-srcInfosCount-10563", device,
-                         build_info_loc.dot(Field::srcInfosCount), "(0x%" PRIx64 ") must be aligned to 256 bytes", build_info.srcInfosCount);
+                         build_info_loc.dot(Field::srcInfosCount), "(0x%" PRIx64 ") must be aligned to 256 bytes",
+                         build_info.srcInfosCount);
+    }
+    return skip;
+}
+
+bool CoreChecks::PreCallValidateCmdBuildClusterAccelerationStructureIndirectNV(
+    VkCommandBuffer commandBuffer, const VkClusterAccelerationStructureCommandsInfoNV *pCommandInfos,
+    const ErrorObject &error_obj) const {
+    bool skip = false;
+    auto cb_state = GetRead<vvl::CommandBuffer>(commandBuffer);
+    const Location command_infos_loc = error_obj.location.dot(Field::pCommandInfos);
+    const LogObjectList objlist(commandBuffer);
+    skip |= ValidateCmd(*cb_state, error_obj.location);
+    skip |= ValidateClusterAccelerationStructureCommandsInfoNV(*pCommandInfos, command_infos_loc);
+    if (!enabled_features.clusterAccelerationStructure) {
+        skip |= LogError("VUID-vkCmdBuildClusterAccelerationStructureIndirectNV-clusterAccelerationStructure-10443", objlist,
+                         error_obj.location, "clusterAccelerationStructures feature was not enabled.");
+    }
+
+    const auto &last_bound_state = cb_state->GetLastBoundRayTracing();
+    const auto *pipeline_state = last_bound_state.pipeline_state;
+    if (pipeline_state && !vku::FindStructInPNextChain<VkRayTracingPipelineClusterAccelerationStructureCreateInfoNV>(
+                              pipeline_state->RayTracingCreateInfo().pNext)) {
+        skip |= LogError("VUID-vkCmdBuildClusterAccelerationStructureIndirectNV-pNext-10444", objlist, error_obj.location,
+                         "The pNext chain of the bound ray tracing pipeline must include a "
+                         "VkRayTracingPipelineClusterAccelerationStructureCreateInfoNV structure.");
```

Comment:
```suggestion
                        "VkRayTracingPipelineClusterAccelerationStructureCreateInfoNV structure.\n%s",
                        PrintPNextChain(Struct::Empty, pipeline_state->RayTracingCreateInfo().pNext).c_str());
```

also want a test for this VU